### ISSUE TITLE
[1.19.3][Forge] Guide book shows White stove over the top of every recipe

### ISSUE THREAD

**Issue by ProsperCraft** [UnlikePaladin/paladin-furniture-mod#39]
![image](https://user-images.githubusercontent.com/4427684/224011196-39f7e204-70b9-4d8e-a35c-1cea18228475.png)

paladin-furniture-mod-1.1.2-forge-mc1.19.3

**Comment by UnlikePaladin** [comment]
Yep, this is a known issue, fixing it in the next version, thanks for the report!

**Comment by jefflima1** [comment]
I lost the book, how can I get it again?

**Comment by UnlikePaladin** [comment]
> I lost the book, how can I get it again?

Hi, it has a crafting recipe, you can craft it in a regular crafting table/or your inventory, you'll need a normal book, 1 orange dye and an iron ingot, it's a shapeless recipe so order doesn't matter.

**Comment by VinceKezel** [comment]
Still says White Stove all in the book in the newest version v1.2.0 fabric
